src/main.jsx

```diff
@@ -4,7 +4,7 @@ import App from "./App.jsx";
 import "./index.css";
 
 createRoot(document.getElementById("root")).render(
-    <StrictMode>
```

Comment:
We should leave Strict Mode on in development in order to catch and address these issues early